sycl/include/sycl/ext/oneapi/atomic_ref.hpp

```diff
@@ -32,16 +31,16 @@ namespace oneapi {
 namespace detail {
 
 // Import from detail:: into ext::oneapi::detail:: to improve readability later
-using namespace ::cl::sycl::detail;
+using namespace ::__sycl_ns::detail;
```

Comment:
@romanovvlad, should we use `::__sycl_ns` instead of `__sycl_ns`?
I think there should be one variant across the whole library. Right?